django-backend/fecfiler/committee_accounts/utils.py

```diff
@@ -34,31 +34,6 @@ def get_response_for_bad_committee_source_config():
     return response
 
 
-def check_email_allowed_to_create_committee_account(email):
```

Comment:
I think we still want the retain the functionality for requiring a fec.gov email in this function, correct?.  I think only functionality we're removing would be the check on line 49.

Reply:
@dheitzer great question! We do not want to keep the requirement that users have an FEC email address.